packager/packager.cc

```diff
@@ -209,6 +209,8 @@ class PackageInfoImpl final : public core::packages::PackageInfo {
     // Whether `visible_to` directives should be ignored for test code
     bool visibleToTests_ = false;
 
+    optional<core::packages::StrictDependenciesLevel> strictDependenciesLevel = nullopt;
```

Comment:
We are likely to want this to track a `LocOffsets` as well.

You could already make use of it in this PR to `addErrorLine` on the previous definition in the case of multiple definitions.

There are other errors we'll be able to use it for in the future.

Personally, I would store the location of the argument, not the whole send, because we might want to do something like change the level of a package in an autocorrect.